virtcontainers/pkg/oci/utils.go

```diff
@@ -861,7 +861,10 @@ func ContainerConfig(ocispec specs.Spec, bundlePath, cid, console string, detach
 		Mounts:      containerMounts(ocispec),
 		DeviceInfos: deviceInfos,
 		Resources:   *ocispec.Linux.Resources,
-		Spec:        &ocispec,
+
+		// This is a custom OCI spec modified at SetEphemeralStorageType()
+		// to support ephemeral storage and k8s empty dir.
+		Spec: &ocispec,
```

Comment:
Idea: Could we also rename this to make it "stand out" a little more? (`CustomSpec: &ocispec,`?)

Reply:
done, thanks